ovirtlago/__init__.py

```diff
@@ -210,15 +210,24 @@ def _create_rpm_repository(
         repo_names,
         custom_sources=None,
         projects_list=None,
+        dist_out=None,
```

Comment:
this name is misleading, the param that's called 'dists' is for 'distributions', as in fc23/el7. While here dist_out dose seem to be an output path?